### DISCUSSION TITLE
Estimating retention factors (Task 2.5)

### DISCUSSION THREAD

**Discussion by JamesSample** [NIVANorge/teotil3#3] [Ideas]
We need to derive approximate retention coefficients for DIN, TON, TDP, TPP, SS and TOC. The retention factors for each individual lake can then be estimated based on residence time (Task 2.4) using Vollenweider.

What options do we have for estimating retention coefficients? Do we need to do this purely based on the literature, or can we compile data for Norwegian lakes where both the inflow and outflow fluxes are measured?

I don't have a good overview of lakes with suitable data. Any suggestions @LeahJB, @okaste? Or alternative approaches?

(To be discussed in tomorrow's meeting...)

**Comment by JamesSample** [comment]
References relating to retention (see e-mail from Øyvind received 03.06.2022 at 08:25):

> Here are a few references on in-lake and in-stream retention based on input-output budgets. Mostly related to N.
> 
> Kelly, C. A., Rudd, J. W. M., Hesslein, R. H., Schindler, D. W., Dillon, P. J., Driscoll, C. T., ... & Hecky, R. E. (1987). Prediction of biological acid neutralization in acid-sensitive lakes. Biogeochemistry, 3(1), 129-140.
> 
> Seitzinger, S. P., Styles, R. V., Boyer, E. W., Alexander, R. B., Billen, G., Howarth, R. W., ... & Breemen, N. V. (2002). Nitrogen retention in rivers: model development and application to watersheds in the northeastern USA. Biogeochemistry 57/58: 199–237, 2002.
> 
> Dillon, P. J., & Molot, L. A. (1990). The role of ammonium and nitrate retention in the acidification of lakes and forested catchments. Biogeochemistry, 11(1), 23-43.
> 
> Dillon, P. J., & Molot, L. A. (1997). Dissolved organic and inorganic carbon mass balances in central Ontario lakes. Biogeochemistry, 36(1), 29-42.
> 
> Kaste, Ø., & Dillon, P. J. (2003). Inorganic nitrogen retention in acid‐sensitive lakes in southern Norway and southern Ontario, Canada—a comparison of mass balance data with an empirical N retention model. Hydrological Processes, 17(12), 2393-2407.

**Comment by LeahJB** [comment]
# Quick summary of discussion with Øyvind, 17.08.2022:

First a chat about different approaches for the different variables/fractions we are interested in:

- DIN: lots of literature on this to base values on. In rivers, can be mostly thought of as nitrate. But lumping with ammonium means we don't need to take care of nitrification of e.g. ammonium inputs from sewage.
- TON: strong correlation with TOC means that we can base the retention factor for TON on the TOC factor, using typical TOC/TON ratios. Then just need to estimate these ratios. E.g. from referanseelva and elveovervåking data (maybe overlap here with work on background inputs?).
- TDP and PP: need to think about this more, as loss of TP from lakes is through burial of PP. So I'm not sure how to deal with TDP. Lots of work has been done on this, so should be a good literature basis. It maybe that we can't separate TP sensibly in Teotil.
- SS: base on simple settling models
- TOC: good data from e.g. Canada. Also some studies from Sweden (see Øyvind's latest reference list).

For PP, the retention coefficient could also be related to SS. Based no SS/PP ratios in monitoring data. Elveovervåkingen has PP and TDP data from 2017 (or 2018?), referanseelver from 2017.

I already suggested 3 possible lakes where we could think about calculating input-output budgets for various elements:
- Langtjern
- Vansjø
- Glitrevann (Drammen area)

Øyvind also mentioned some other possibilities:
- Mjøsa (lots of work has been done here. Too big a job to re-do the mass balance, but ask Jan Erik or Anne Lyche if N and P budgets have been done, and for references)
- Halden vassdraget: he thinks there is a lake there where nutrient retention will have been looked at. Ask Sigrid H.
- Byglandsfjorden (acid rain monitoring)
- Herrefossfjorden (acid rain monitoring). Two inflows and the outflow are monitored for chem. Short residence time.
- Perhaps Maridalsvannet, given its importance for drinking water?

Generally, we agreed that it would be a good idea to start from the literature with a broad-brush review, to get some basic estimates, before getting bogged down in data. Calculating mass balance for lakes can quickly become messy and time-consuming, especially if you don't have perfect monitoring data of discharge and concentrations from both the inflow and outflow, and you need a long-enough time series to be able to generalise.

**Comment by LeahJB** [comment]
## Additional references

Provided by Øyvind, 11.08.2022:

### P retention models (Vollenwider type) adapted to Norwegian conditions

P retention in large, deep lakes: 
Rognerud, S., Berge, D. og Johannessen, M. 1979. Telemarksvassdraget – Hovedrapport fra undersøkelsene i perioden 1975-1979. NIVA-rapport 1147, 83 s.
Based on: 
Larsen DP, Mercier HT. 1976. Phosphorus retention capacity of lakes. J. Fish. Res. Borad. Can. 33: 1742-1750
P retention in shallow lakes:
Berge, D. 1987. Fosforbelastning og respons i grunne og middels grunne innsjøer. NIVA-rapport 2001, 51 s.

### Other references (mostly Swedish studies)
Andersson L, Rosberg J, Pers BC, Olsson J, Arheimer B. 2005. Estimating catchment nutrient flow with the HBV-NP model – sensitivity to input data. Ambio 34: 521-532.
Anonymous 2014. Description of the HYPE model (version 4.8.0). Dated 2014-05-14, 55 pp.
Arheimer B, Brandt M. 1998. Modeling nitrogen transport and retention in the catchments of southern Sweden. Ambio 27: 471–480
Arheimer B, Dahné J, Donnelly C, Lindström G, Strömqvist J. 2012. Water and nutrient simulations using the HYPE model for Sweden vs. the Baltic Sea basin – influence of input-data quality and scale. Hydrology research 43: 315-329.
Arheimer B, Pers C. 2007. Kväveretention i svenska sjöar och vattendrag – betydelse för utsläpp från reningsverk (in Swedish). SMHI Hydrologi, report no. 107, 62 pp.
Billen G, Silvestre M, Grizzetti B, Leip A, Garnier J, Voss M, Howarth R, Bouraoui F, Lepisto A, Kortelainen P, Johnes P, Curtis C, Humborg C, Smedberg E, Kaste Ø, Ganeshram R, Beusen A, Lancelot C. 2011. Nitrogen flows from European regional watersheds to coastal marine waters. In: Sutton MA et al. [Eds]. The Europe Nitrogen Assessment – Sources, Effects and Policy perspectives. Cambridge University Press, pp. 271-297.
Darracq A, Destouni G. 2005. In-Stream Nitrogen Attenuation: Model-Aggregation Effects and Implications for Coastal Nitrogen Impacts. Environ. Sci. Technol. 39: 3716-3722
Darracq A, Lindgren G, Destouni G. 2008. Long-term development of phosphorus and nitrogen loads through the subsurface and surface water systems of drainage basins. Global Biogeochemical Cycles 22: GB3022
Durand P, Breuer L, Johnes PJ, Billen G, Butturini A, Pinay G, van Grinsven H, Garnier J, Rivett M, Reay DS, Curtis C, Siemens J, Maberly S, Kaste Ø, Humborg C, Loeb R, de Klein J, Hejzlar J, Skoulikidis N, Kortelainen P, Lepistö A, Wright RF. 2011. Nitrogen processes in aquatic ecosystems. In: Sutton, M.A et al. [Eds]. The Europe Nitrogen Assessment – Sources, Effects and Policy perspectives. Cambridge University Press, pp. 126-146.
Ejhed H, Olshammar M, Brånvall G, Gerner A, Bergström J, Johnsson H, Blombäck K, Nisell J, Gustavsson H, Persson C, Alavi G. 2011. Beräkning av kväve- och fosforbelastning på vatten och hav för uppföljning av miljökvalitetsmålet ”Ingen övergödning”. SMED report no. 56, 55 pp.
Grimvall, A., von Brömssen, C. & Lindström, G. (2014) Using process-based models to filter out natural variability in observed concentrations of nitrogen and phosphorus in river water. Environmental Monitoring and Assessment, 186:5135-5152. doi: 10.1007/s10661-014-3765-y
Gustavsson H, Tengdelius Brunell J, Alavi G. 2011. Retentionsberäkning för enskilda reningsverk i Södra Sverige (in Swedish). SMHI report nr 2011-57, 21 pp.
Gustavsson H, Westerberg I, Widén-Nilsson E. 2012. Osäkerhetsanalys av kväveretention i HBV-NP-modellen (in Swedish). SMED Rapport  nr 101/ 2012, 67 pp.
Köhler SJ, Kothawala D, Futter MN, Liungman O, Tranvik L. 2013. In-lake processes offset increased terrestrial inputs of dissolved organic carbon and color to lakes. PLoS ONE 8: e70598. doi:10.1371/journal.pone.0070598
Kronvang B, Borgvang SA, Barkved LJ. 2009. Towards European harmonised procedures for quantification of nutrient losses from diffuse sources—the EUROHARP project. J. Environ. Monit. 11: 503–505
Lindström G, Pers CP, Rosberg R, Strömqvist J, Arheimer B. 2010. Development and test of the HYPE (Hydrological Predictions for the Environment) model – A water quality model for different spatial scales. Hydrology Research 41: 295-319.
Lindström G, Rosberg J, Arheimer B. 2005 Parameter precision in the HBV-NP model and impacts on nitrogen scenario simulations in the Rönneå River, Southern Sweden. Ambio 34: 533–537.
Schoumans OF, Silgram MP, Groenendijk P, et al. 2003. Description of nine nutrient loss models: capabilities and suitability based on their characteristics. Journal of Environmental Monitoring 11: 506 –514.
Seitzinger SP. 1988. Denitrification in freshwater and coastal marine ecosystems: Ecological and geochemical significance. Limnol. Oceanogr. 33: 702-724.
SMHI report 2016-12-21. Kommunala avloppsreningsverk. Utsläpp, rinntid och retention.
Strömqvist J, Arheimer B, Dahné J, Donnelly C, Lindström G. 2012. Water and nutrient predictions in ungauged basins – Set-up and evaluation of a model at the national scale. Hydrological Sciences Journal 57: 229-247.
Tengdelius Brunell J, Gustavsson H, Dahné J, Alavi G. 2016. Retention beräknad med SMED-HYPE. Underlagsrapport till Pollution Load Compilation 6. SMED Rapport Nr 190/2016 (in Swedish)
Tengdelius Brunell J, Person H, Nilsson J, Blombäck K, Johnsson H. 2013. Anpassning av HYPE-modellen för läckage-koefficienter och typhalter. SMED Rapport Nr 114 (in Swedish)
Wilander A, Persson G. 2001. Recovery from Eutrophication: Experiences of Reduced Phosphorus Input to the Four Largest Lakes of Sweden. AMBIO 30: 475-485.

**Reply by LeahJB** [reply]
## Additional references from Heleen, relating to TOC removal, 10.08.2022:

Paper on Langtjern: https://aslopubs.onlinelibrary.wiley.com/doi/pdfdirect/10.1002/lol2.10077  has references on lake C retention (start of discussion). Francois also estimated lake DOC retention but he used a slightly different input discharge and came to a higher % than 8%. https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2021JG006359 

Not a lot from Norway, but boreal lakes in Sweden and Finland are equally relevant. Valkea-Kotinen, there I know they have inputs and outputs, and probably a longer residence time. 
Then there is the Catalan et al. (2016) Nat. Geosci. paper, attached – that is about decay rates, but perhaps something useful there.
